features/payment/managing_payment_methods/sorting_payment_methods.feature

```diff
@@ -15,37 +15,37 @@ Feature: Sorting listed payment methods
         And this payment method is named "Płatność Przy Odbiorze" in the "Polish (Poland)" locale
         And I am logged in as an administrator
 
-    @ui
+    @ui @api
     Scenario: Payment methods can be sorted by their codes
         Given I am browsing payment methods
         When I start sorting payment methods by code
         Then I should see 3 payment methods in the list
         And the first payment method on the list should have code "cash_on_delivery"
 
-    @ui
+    @ui @api
     Scenario: Changing the order of sorting payment methods by their codes
         Given I am browsing payment methods
         And the payment methods are already sorted by code
-        When I switch the way payment methods are sorted by code
+        When I switch the way payment methods are sorted descending by code
```

Comment:
`are sorted descending by code` sounds awkward to my ear.

I don't have better phrasing, but here are ideas,

```suggestion
        When I switch the way payment methods are sorted to descending by code
```
```suggestion
        When I switch the way payment methods are sorted by descending for code
```